MAGFit/magfit.js

```diff
@@ -2010,7 +2010,7 @@ function load(log_file) {
             continue
         }
 
-        const tip_text = "Fits with no motor compensation.<ul><li>Offsets: Fit of only X Y Z offset parameters.</li><li>Offsets and scale: Fit of X Y Z offsets and single scale factor.</li><li>Offsets and iron: Fit of X Y Z offsets and iron compensation matrix diagonals and off-diagonals.</li></ul>"
+        const tip_text = "Calibrations with no motor compensation.<ul><li>Offsets: Fit of only X Y Z offset parameters.</li><li>Offsets and scale: Fit of X Y Z offsets and single scale factor.</li><li>Offsets and iron: Fit of X Y Z offsets and iron compensation matrix diagonals and off-diagonals.</li></ul> If a calibration cannot be selected the tool was unable to find a valid solution, a longer flight with better coverage will give a better chance of finding a solution."
```

Comment:
There still one "Fit" here in this line.. in case that was unintentional..

Reply:
Well spotted, fixed.